autoexec.cfg

```diff
@@ -33,42 +33,44 @@ sar_hud_position 2
+sar_function mw_up_rest restore "bind mwheelup"
+
 // Dialogue toggle
-sar_hud_set_text 0 "Dialogue OFF"
-sar_hud_set_text 1 "Dialogue ON"
-sar_hud_set_text_color 0 #FF5555
-sar_hud_set_text_color 1 #55FF55
-sar_alias dialogue_off "sar_hud_show_text 0; sar_hud_hide_text 1; sar_alias dialogue_toggle dialogue_on;  snd_setmixer potatosVO vol 0.01; snd_setmixer gladosVO vol 0.01; snd_setmixer announcerVO vol 0.01; snd_setmixer wheatleyVO vol 0.01; snd_setmixer caveVO vol 0.01; snd_setmixer coreVO vol 0.01"
-sar_alias dialogue_on  "sar_hud_show_text 1; sar_hud_hide_text 0; sar_alias dialogue_toggle dialogue_off; snd_setmixer potatosVO vol 0.4;  snd_setmixer gladosVO vol 0.7;  snd_setmixer announcerVO vol 0.7;  snd_setmixer wheatleyVO vol 0.7;  snd_setmixer caveVO vol 0.88; snd_setmixer coreVO vol 0.75"
+sar_function snd_setdialogue "snd_setmixer potatosVO vol $1; snd_setmixer gladosVO vol $2; snd_setmixer announcerVO vol $2; snd_setmixer wheatleyVO vol $2; snd_setmixer caveVO vol $3; snd_setmixer coreVO vol $4"
+sar_alias dialogue_off       "sar_hud_set_text 0 #FF5555Dialogue OFF; sar_alias dialogue_toggle dialogue_on;  snd_setdialogue .01 .01 .01 .01"
+sar_alias dialogue_on        "sar_hud_set_text 0 #55FF55Dialogue ON;  sar_alias dialogue_toggle dialogue_off; snd_setdialogue .4; .7; .88 .75"
 dialogue_on
 
-// Funneling toggle
-sar_hud_set_text 2 "Funneling OFF"
-sar_hud_set_text 3 "Funneling ON"
-sar_hud_set_text_color 2 #FF5555
-sar_hud_set_text_color 3 #55FF55
-sar_alias funneling_off "sar_hud_show_text 2; sar_hud_hide_text 3; sar_alias funneling_toggle funneling_on;  sv_player_funnel_into_portals 0"
-sar_alias funneling_on  "sar_hud_show_text 3; sar_hud_hide_text 2; sar_alias funneling_toggle funneling_off; sv_player_funnel_into_portals 1"
-funneling_on
-
 // Fullbright toggle
-sar_alias fullbright_on  "sar_alias fullbright_toggle fullbright_off; mat_ambient_light_r 0.1; mat_ambient_light_g 0.1; mat_ambient_light_b 0.1"
-sar_alias fullbright_off "sar_alias fullbright_toggle fullbright_on;  mat_ambient_light_r 0.0; mat_ambient_light_g 0.0; mat_ambient_light_b 0.0"
+sar_function ambient_rgb "mat_ambient_light_r $1; mat_ambient_light_g $1; mat_ambient_light_b $1"
+sar_alias fullbright_on  "sar_alias fullbright_toggle fullbright_off; ambient_rgb .1"
+sar_alias fullbright_off "sar_alias fullbright_toggle fullbright_on;  ambient_rgb 0"
 fullbright_off
 
 // Sensitivity toggle
-sar_alias sensitivity_on  "sar_alias sensitivity_toggle sensitivity_off; sensitivity 0.1"
-sar_alias sensitivity_off "sar_alias sensitivity_toggle sensitivity_on;  sensitivity 2"
-sensitivity_off
+sar_function sensitivity_reset   "sensitivity $sensitivity"
+sar_function sensitivity_capture "svar_from_cvar sensitivity sensitivity"
+sar_alias sensitivity_on  "sar_alias sensitivity_toggle sensitivity_off; sensitivity_capture; sensitivity 0.1"
+sar_alias sensitivity_off "sar_alias sensitivity_toggle sensitivity_on;  sensitivity_reset"
+sensitivity_capture; sensitivity_off
```

Comment:
I did something like this before I looked at this PR, great minds think alike I suppose :p